controller/articles.controller.js

```diff
@@ -0,0 +1,21 @@
+const { fetchArticleById } = require('../models/articles.model');
+
+exports.getArticleById = (req, res, next) => {
+  const { article_id } = req.params
+
+ 
+  if (isNaN(article_id)) {
+    return res.status(400).send({ msg: 'invalid article id' })
+  }
+
+  fetchArticleById(article_id)
+    .then((article) => {
+      if (!article) {
```

Comment:
We should only be handling errors within a .catch otherwise it defeats the purpose of having .then() and .catch() - also consider how many times you'll need to send back these status codes and messages - we could now set up a custom error handler to do this for us!